packaging/helm-charts/helm3/strimzi-kafka-operator/templates/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -29,9 +29,14 @@ spec:
       {{- end }}
     spec:
       serviceAccountName: {{ .Values.serviceAccount }}
-      {{- if .Values.image.imagePullSecrets }}
+      {{- if kindIs "string" .Values.image.imagePullSecrets }}
```

Comment:
Maybe you can add here some comment that this is here for backwards compatibility reasons? We might forget it in few months / years and wonder why both options are there.

Reply:
Added